### PR TITLE
added condition to prevent error when we don't upload any file

### PR DESCRIPTION
**Pull request by jenachoi**
This is necessary since we will add nested form for updating template division. The only change in the code here is adding 'unless' statement to prevent the error that will occur when we update 'only' template division.

### CODE REVIEW

File: app/controllers/exam_templates_controller.rb [hunk 1]
```diff
@@ -20,18 +20,21 @@ def download
...
-    old_template_filename = old_exam_template.filename
-    old_exam_template.replace_with_file(new_uploaded_io.read, assignment_id: assignment.id, filename: old_template_filename)
-    flash_message(:success, t('exam_templates.update.success'))
+    # updating exam template file
+    new_uploaded_io = params[:exam_template][:new_template]
+    unless new_uploaded_io.nil?
+      # error checking when new_uploaded_io is not pdf
+      if new_uploaded_io.content_type != 'application/pdf'
+        flash_message(:error, t('exam_templates.update.failure'))
+        redirect_to action: 'index'
```
Comment (david-yz-liu) [inline app/controllers/exam_templates_controller.rb]: Don't put the `redirect_to` (or `return`) here. Instead, put the following code in an `else` block after this, and then keep the `redirect_to` on line 38.

**Comment by david-yz-liu** [comment]
@jenachoi I'm not sure the problem is fixed (in particular, `params[:exam_template]` might be nil. Also, there's a conflict now that I merged in the responders branch, so please do a merge and then fix this as needed.

**Comment by jenachoi** [comment]
I haven't finished this yet!

**Comment by jenachoi** [comment]
finished- Please let me know if there is any changes needed

**Pull request merged**